lib/parity/environment.rb

```diff
@@ -57,9 +57,9 @@ def deploy_to_heroku
     end
 
     def restore
-      if production?
+      if production? && additional_restore_arguments["--force"].nil?
```

Comment:
Regex probably needs to be slightly more robust so we're checking only for `--force` and not something that contains `--force`.